app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -594,8 +593,7 @@
   <string name="transfer_rate">傳輸速率</string>
   <string name="time_remaining">剩餘時間</string>
   <string name="grant_apkinstall_permission">程式需要安裝程式的權限以繼續。</string>
-  <string name="internalstorage">內部儲存空間</string>
-  <string name="archive_password_prompt">請輸入壓縮檔的密碼。</string>
+    <string name="archive_password_prompt">請輸入壓縮檔的密碼。</string>
```

Comment:
Here and in most other files, indentation is mismatched

Reply:
Doesnt matter.